connector/ldap/ldap_test.go

```diff
@@ -123,6 +123,80 @@ userpassword: bar
 	runTests(t, schema, connectLDAP, c, tests)
 }
 
+func TestQuerySuffix(t *testing.T) {
```

Comment:
Thanks for adding this test -- would you mind cutting it down a little?

I'd think two cases would be enough: the happy path, and one with the `EmailAttr` missing 🤔 Invalid password, and not-found-user don't seem required there, it's not related to the to email suffix.

[nit] Let's add Email there: 
```suggestion
func TestQueryWithEmailSuffix(t *testing.T) {
```